homeworks/andrii.kholodenko_AndriiCold/4-modules/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Data</title>
```

Comment:
```suggestion
        <title>Date</title>
```
Data це дані, а Date це дата